src/controller/Delete.ts

```diff
@@ -0,0 +1,11 @@
+
+import * as express from 'express';
+var delet = express.Router();
+
+
+// get route
+delet.get('/', function (req: express.Request, res: express.Response) {
+  res.send('delete');
+})
+
+module.exports = delet;
```

Comment:
do you want this to be "module.exports = delet**e**"?

Reply:
No because the word "delete" is a reserved word so I had to use a different name